### PR TITLE
Description field for gene panels

### LINKED ISSUE #1308: "Description" field on gene panel overview
It would be good to have a free text field with a label "Description" or similar to write more general information regarding each panel.

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #1308 

- Panel description can be optionally provided when creating a new panel:

image

- Description can be alternatively modified or added from the panel page:

image

**How to test**
Install the branch on clinical-db stage:
```
ssh your.user@clinical-db.scilifelab.se
sudo -iu hiseq.clinical
cd servers/resources/clinical-dbscilifelab.se
bash update-scout-stage.sh panel_description
```

1. Try to create a new gene panel with and without description (tip: a csv file with a gene list is in the scout/demo data folder of scout)

1. Make sure that you can modify ( or cancel ) the panel description from the gene panel page.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by MM
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Looks like OMIM server is not responsive. I'm changing the tests to take care of the eventuality

**Comment by northwestwitch** [comment]
@henrikstranneheim I've fixed the problem of the topbar navigation in this PR

**Comment by henrikstranneheim** [comment]
I saw: ⭐️

**Comment by northwestwitch** [comment]
Whoever has time, it's an easy review! :)

File: scout/server/blueprints/panels/controllers.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def update_panel(store, panel_name, csv_lines, option):
     return panel_obj
 
 
-def new_panel(store, institute_id, panel_name, display_name, csv_lines):
+def new_panel(store, institute_id, panel_name, display_name, description, csv_lines):
```
Comment (moonso) [inline scout/server/blueprints/panels/controllers.py]: Is this really a mandatory argument?
```suggestion
def new_panel(store, institute_id, panel_name, display_name, csv_lines, description=None):
```

File: scout/server/blueprints/panels/controllers.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def update_panel(store, panel_name, csv_lines, option):
...
     """Create a new gene panel.
 
     Args:
```
Comment (moonso) [inline scout/server/blueprints/panels/controllers.py]: Could you update the docstring as well?

File: tests/server/blueprints/panels/test_panels_views.py [hunk 3]
```diff
@@ -28,6 +28,27 @@ def test_panel(client, real_panel_database, panel_info):
     assert resp.data.count('href="/genes/'.encode()) == len(panel_obj['genes'])
 
 
+def test_panel_update_description(client, real_panel_database):
```
Comment (moonso) [inline tests/server/blueprints/panels/test_panels_views.py]: 👍

File: scout/server/blueprints/panels/controllers.py (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/panels/controllers.py]: Done

**Pull request merged**